picard/ui/options/dialog.py

```diff
@@ -137,7 +143,11 @@ def __init__(self, default_page=None, parent=None):
         self.finished.connect(self.saveWindowState)
 
         for page in self.pages:
-            page.load()
+            try:
+                page.load()
+            except Exception:
+                log.error('Failed loading options page %r', page, exc_info=True)
```

Comment:
This can just be log.exception and you don't need to pass the exc_info

Reply:
Indeed, I changed it